_docs/_latest/configure/gsuite-group-collection.md

```diff
@@ -15,10 +15,6 @@ service account just for this functionality. Read more about
 
 ### Create a service account
 
-**Note:** If you used the setup wizard to setup Forseti, it already creates a G Suite 
```

Comment:
I believe so since we're not creating a specific gsuite service account anymore with @ahoying 's fix. We'll still want to review this doc for accuracy

Reply:
You are right, I was not thinking about Andrew's fix.